recipe/meta.yaml

```diff
@@ -73,8 +70,8 @@ requirements:
     # pick it up.
     - {{ pin_compatible('gflags', max_pin='x.x') }}  # [not win]
     # brotli doesn't have run_exports defined.
-    - brotli  # [osx]
-    - utf8proc
+    - pin_compatible('brotli', max_pin='x') # [osx]
```

Comment:
brotli defines a run_exports like `x.x`, so unless you `ignore_run_exports` it, the run_exports will win (because it's more strict).

Reply:
Charles' comment was for `utf8proc`.

Reply:
I mean line 72:
# brotli doesn't have run_exports defined.

Reply:
This comment is right brotli does have a run_exports. See https://conda-metadata-app.streamlit.app/?q=conda-forge%2Flinux-64%2Fbrotli-1.1.0-hd590300_1.conda.